tests/scripts/all.sh

```diff
@@ -2468,6 +2468,131 @@ component_test_psa_crypto_config_accel_ecc () {
     make test
 }
 
+# Helper function used in:
+# - component_test_psa_crypto_config_accel_all_curves_except_p192
+# - component_test_psa_crypto_config_accel_all_curves_except_x25519
+# to build and test with all accelerated curves a part from the specified one.
+psa_crypto_config_accel_all_curves_except_one () {
+    BUILTIN_CURVE=$1
+
+    msg "build: PSA_CRYPTO_CONFIG + all accelerated EC algs (excl secp192r1) + USE_PSA_CRYPTO"
+
+    # Accelerate all EC algs (all EC curves are automatically accelerated as
+    # well in the built-in version due to the "PSA_WANT_xxx" symbols in
+    # "crypto_config.h")
```

Comment:
Minor: we could get rid of this parenthesis by just adding the curve heres, which seems a bit more logical to me: the top-level list is for things that we want to be present in libtestdriver and accelerated in the main libraries, which is the case for curves here. Either 
```
   loc_accel_list="ALG_ECDH \
                    ALG_ECDSA ALG_DETERMINISTIC_ECDSA \
                    ALG_JPAKE \
                    KEY_TYPE_ECC_KEY_PAIR KEY_TYPE_ECC_PUBLIC_KEY \
                    ECC_BRAINPOOL_P_R1_256 ECC_BRAINPOOL_P_R1_384 ECC_BRAINPOOL_P_R1_512 \
                    ECC_MONTGOMERY_255 ECC_MONTGOMERY_448 \
                    ECC_SECP_K1_192 ECC_SECP_K1_224 ECC_SECP_K1_256 \
                    ECC_SECP_R1_192 ECC_SECP_R1_224 ECC_SECP_R1_256 ECC_SECP_R1_384 ECC_SECP_R1_521"
```
or using sed and a for loop to enumerate and add the curves, as discussed in another comment.

Wdyt?

Reply:
Actually, when building the driver we want to have the built-in version of all the curves supported (since there is no other lower lever SW which can provide them), so according to `config_psa.h` we want:

- `PSA_WANT_ECC_` and NOT `MBEDTLS_PSA_ACCEL_ECC_` --> `MBEDTLS_PSA_BUILTIN_ECC_`

In this case the `crypto_config_test_driver_extensions.h` do no perform any "adaptation" for the curves' defines.

In short: I would keep the current declaration for curves where it is now. Is it wrong?